doc/nrf/config_and_build/programming.rst

```diff
@@ -88,7 +90,7 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <a
 
          west flash --erase
 
-      This command clears the full flash memory before programming, which is the recommended approach.
+      This command erases the full flash memory before programming, which is the recommended approach.
```

Comment:
NIT: `west` supports also one pretty good feature that allows users to flash the device without checking and rebuilding the code. (without that after calling `west flash` the build system checks if there are no changes and then rebuilds if so, and flashes the device). To skip checking and rebuilding we can add the `--skip-rebuild` argument to the `west flash` command. Of course, using it requires some attention from the user, and they need to know about the consequences, but in many cases, it can be helpful. Of course, it is described in the `west` documentation, but I wonder whether we can add it here. If not in the scope of this PR - please skip my wondering.

Reply:
Added this for consideration to https://nordicsemi.atlassian.net/browse/NCSDK-22324. Thanks, @ArekBalysNordic !